LayoutTests/imported/w3c/web-platform-tests/fetch/api/body/formdata.any.js

```diff
@@ -12,3 +12,14 @@ promise_test(async t => {
   const fd = await req.formData();
   assert_true(fd instanceof FormData);
 }, 'Consume empty request.formData() as FormData');
+
+promise_test(async t => {
```

Comment:
This test is probably new so we should upstream it (or at least file the PR) to WPT before landing this one.

Reply:
Thanks yeah I will export it right now

Reply:
Opened https://github.com/web-platform-tests/wpt/pull/42455 for the WPT upstreaming